gh2/csv.py

```diff
@@ -245,14 +251,39 @@ def main():
         'Milestone'
```

Comment:
this isn't a major concern right now, but can be added as a feature if needed later.